docs/book/beyond_basics/using_client_go_informers.md

```diff
@@ -0,0 +1,68 @@
+
+# Using Generated Informers
+
+This chapter describes how to use the client-go generated Informers with controller-runtime Watches.
+
+[Link to reference documentation](https://godoc.org/sigs.k8s.io/controller-runtime/pkg/source#Informer)
+
+**Note:** The `source.Informers` and `source.Kind` sources create different caches.  Using both
+`source.Informers` and `source.Kind` sources in the same project will result in duplicate caches.
```

Comment:
we can probably make it more clear that it will not impact the functionality, but double the memory required by the controller.

Reply:
Added